examples/project_configuration/webhooks.ipynb

```diff
@@ -18,8 +18,7 @@
                 "    2. label_updated\n",
```

Comment:
<p>Maybe we should clear output here too</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>